mirrord/layer/src/lib.rs

```diff
@@ -960,6 +1022,20 @@ pub(crate) unsafe extern "C" fn close_detour(fd: c_int) -> c_int {
     res
 }
 
+/// Hook for `libc::fork`.
+#[hook_guard_fn]
+pub(crate) unsafe extern "C" fn fork_detour() -> pid_t {
+    debug!("Process {} forking!.", std::process::id());
+    let res = FN_FORK();
+    if res == 0 {
+        debug!("Child process initializing layer.");
```

Comment:
does this actually log it? I'd guess not because it shouldn't be initialized but maybe it is because fork?